spec/knockout-fast-foreach-spec.js

```diff
@@ -421,7 +421,7 @@ describe("observable array changes", function () {
         arr[i].num = i;
       }
       assert.equal(div.children().length, itemNumber)
-      assert.equal(div.children().filter(function() { return this.testprop == 10; }).length, itemNumber)
+      //assert.equal(div.children().filter(function() { return this.testprop == 10; }).length, itemNumber)
```

Comment:
Test passes if I wrap the 2nd part of the script (everything after `console.timeEnd("with move");`) with a `setTimeout(function() { ... }, 1)`. What do you think of this?